addons/medical_statemachine/stringtable.xml

```diff
@@ -4,33 +4,43 @@
         <Container name="Settings">
             <Key ID="STR_ACE_Medical_statemachine_subCategory">
                 <English>States</English>
+                <German>Zustände</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryCondition_DisplayName">
                 <English>Fatal Injury Player</English>
+                <German>Tötliche Spielerverletzungen</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryCondition_Description">
                 <English>Defines when Player can receive a fatal injury</English>
+                <German>Definiert wann ein Spieler tötliche Verletzungen bekommen kann.</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryConditionAI_DisplayName">
                 <English>Instant death AI</English>
+                <German>Direkter Tot KI</German>
```

Comment:
"Tod" is a noun here. "tot" is adjective.
```suggestion
                <German>Direkter Tod KI</German>
```
Don't really like that translation. Maybe "von KI" ?